src/Numerics/DGmethods/DGmodel_kernels.jl

```diff
@@ -193,6 +191,66 @@ Computational kernel: Evaluate the volume integrals on right-hand side of a
                 end
             end
 
+            # In the case of the remainder model we may need to loop through the
```

Comment:
My internal monologue here (just comments for reference and make sure I understand these new lines of code): lines 194--253 simply just check for components operating in a specific `Direction`, which then the `local_flux` (in that direction) is initialized (to zero) and filled appropriately.

Reply:
Not sure I understand the question.

The point of this is that if a subcomponent operates in only a subset of the directions we need to handle those subcomponent directions separately, since horizontal and vertical is defined with the reference element so we need to for each direction:

- Evaluate in physical space
- Rotate to reference state
- Add in the rotated values.

Reply:
Not really a question; just checking what is going on here!